spark/v3.5/spark/src/test/java/org/apache/iceberg/spark/CatalogTestBase.java

```diff
@@ -48,6 +46,4 @@ protected static Object[][] parameters() {
       }
     };
   }
-
-  @TempDir protected Path temp;
```

Comment:
there's already a `temp` variable available in the super class (`TestBaseWithCatalog`), so we don't need to have 2 different ones